src/mcp_agent/mcp/mcp_aggregator.py

```
@@ -122,103 +126,130 @@
+            span.set_attribute("force", force)
+            span.set_attribute("connection_persistence", self.connection_persistence)
+            span.set_attribute(GEN_AI_AGENT_NAME, self.agent_name)
+            span.set_attribute("initialized", self.initialized)
+
+            if self.initialized and not force:
+                return
+
+            # Keep a connection manager to manage persistent connections for this aggregator
+            if self.connection_persistence:
+                # Try to get existing connection manager from context
+                # TODO: saqadri (FA1) - verify
```

Pylint (3.3.7)

[warning] TODO: saqadri (FA1) - verify

(W0511)